eman2/protocols/protocol_tomo_boxing.py

```diff
@@ -82,6 +88,45 @@ def _readSetOfCoordinates3D(self, jsonBoxDict, tomo, coord3DSetDict):
 
                 self._readCoordinates3D(box, tomo, coord3DSet)
 
+    def _coordinates2json(self, inputCoor):
+        cwd = os.getcwd()
+        infoDir = pwutils.join(cwd, 'info')
+        self._leaveWorkingDir()
+        fnInputCoor = 'extra-%s_info.json' % pwutils.removeBaseExt(self.inputTomo.getFileName())
+        pathInputCoor = pwutils.join(infoDir, fnInputCoor)
+        if not exists(pathInputCoor):
+            pwutils.makePath(infoDir)
+        f = open(pathInputCoor, 'w')
+        initFile = '{\n"boxes_3d": [\n\n],\n"class_list": {\n"0": {\n"boxsize": 32,\n"name": "particles_00"\n}\n}\n}'
+        f.write(initFile)
+        f.close()
+        firstItem = inputCoor.getFirstItem()
```

Comment:
What's special about the first item? Can not this logic go inside the loop together with the rest of the item?

Reply:
Solved in https://github.com/scipion-em/scipion-em-eman2/pull/35/commits/1a2b4d12be94ff10faa5b5eac8373d8028d61f89